mobile/src/main/java/ch/epfl/sdp/mobile/ui/play/PlayScreen.kt

```diff
@@ -0,0 +1,50 @@
+package ch.epfl.sdp.mobile.ui.play
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Button
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import ch.epfl.sdp.mobile.state.LocalLocalizedStrings
+
+/**
+ * Composable that composes the PlayScreen [FIXME] Contains a new game button only, should be
+ * expanded
+ *
+ * @param state state of the PlayScreen
+ * @param modifier the [Modifier] for this composable.
+ */
+@Composable
+fun PlayScreen(
+    state: PlayScreenState,
+    modifier: Modifier = Modifier,
+) {
+  Box(contentAlignment = Alignment.BottomEnd, modifier = modifier.fillMaxSize()) {
+    NewGameButton(onNewGame = state.onNewGameClick)
+  }
+}
+
+/**
+ * Composable that composes a new game button
+ * @param onNewGame callable upon button action
+ * @param modifier the [Modifier] for this composable.
+ */
+@Composable
+fun NewGameButton(onNewGame: () -> Unit, modifier: Modifier = Modifier) {
+  val strings = LocalLocalizedStrings.current
+  Button(
```

Comment:
You may want to look into the `ExtendedFloatingActionButton` composable : https://developer.android.com/reference/kotlin/androidx/compose/material/package-summary#ExtendedFloatingActionButton(kotlin.Function0,kotlin.Function0,androidx.compose.ui.Modifier,kotlin.Function0,androidx.compose.foundation.interaction.MutableInteractionSource,androidx.compose.ui.graphics.Shape,androidx.compose.ui.graphics.Color,androidx.compose.ui.graphics.Color,androidx.compose.material.FloatingActionButtonElevation).